app/components/initEndpoint/initEndpointController.js

```diff
@@ -27,7 +27,6 @@ function ($scope, $state, EndpointService, StateManager, EndpointProvider, Notif
   $scope.createLocalEndpoint = function() {
     $('#createResourceSpinner').show();
     var name = 'local';
-    var URL = 'unix:///var/run/docker.sock';
 
     var endpointID = 1;
     EndpointService.createLocalEndpoint(name, URL, false, true)
```

Comment:
You can remove the usage of URL here: `EndpointService.createLocalEndpoint(name, false, true)`